docs/book/v3/form-element-manager.md

```diff
@@ -0,0 +1,279 @@
+# Form Element Manager
+
+The form element manager is a *specialized* dependency injection container to obtain objects which implements the `Laminas\Form\ElementInterface` interface.
+
+The manager is the central object of laminas-form to create and retrieve all types of form elements, fieldsets and forms.
+
+It handles all included form elements of laminas-form, user-defined elements and also whole forms.
+
+The manager is based on the [plugin manager of laminas-servicemanager](https://docs.laminas.dev/laminas-servicemanager/plugin-managers/).
+
+## Benefits
+
+The following benefits are provided by the manager:
+
+- Handles all forms, fieldsets and elements, including all custom ones.
+- Create forms, fieldsets and elements with all dependencies like validators and filters.
+- Can create forms, fieldsets and elements with name and options.
+- Can provide elements, fieldsets and forms without prior registration.
+- Fetches hydrators and input filters from the application's service container and add them to a form or a fieldset.
+- Allows to override existing elements or to extend them.
+
+## Create a Form Element Manager
+
+To create an instance, the form element manager requires a PSR-11 dependency container. The following examples uses the container implementation of [laminas-servicemanager](https://docs.laminas.dev/laminas-servicemanager/):
+
+```php
+$formElementManager = new Laminas\Form\FormElementManager(
+    new Laminas\ServiceManager\ServiceManager()
+);
+```
```

Comment:
In practice, users will generally just pull the `FormElementManager` **_from_** the `ServiceManager`, though; this usage here is going to be quite rare, no?